app-web/src/utils/helpers.js

```diff
@@ -165,21 +165,26 @@ export const isFilterLonely = filters => {
  *
  * @param {Array} topics the raw topic data (aka collections) as taken from the graphql query
  * @param {Object} topics[ind] {node: {..., name: {string }}}
- * @returns {Array} the sorted topics with Featured Cards and Design System first!
+ * @returns {Array} the sorted topics with Featured Cards, Design System and Getting Started on the DevOps Platform first!
  */
-export const sortDevhubTopicsAfterDesignSystemAndFeatured = topics => {
-  // it is unknown what position design system and featured cards will be since this can change build to build
+export const sortDevhubTopicsAfterSelectedTopics = topics => {
+  // it is unknown what position the selected topcs will be since this can change build to build
   const featuredIndex = topics.findIndex(topic => topic.node.name === TOPICS.FEATURED_RESOURCES);
   const designIndex = topics.findIndex(topic => topic.node.name === TOPICS.DESIGN_SYSTEM);
-  const designSystem = { ...topics[designIndex] };
-  const featuredResources = { ...topics[featuredIndex] };
-  // remove design system from topics list and then featured cards
-  const topicsWithoutDS = topics.filter(topic => topic.node.name !== TOPICS.FEATURED_RESOURCES);
-  const topicsWithoutDSAndFC = topicsWithoutDS.filter(
-    topic => topic.node.name !== TOPICS.DESIGN_SYSTEM,
+  const devOpsPlatformIndex = topics.findIndex(
+    topic => topic.node.name === TOPICS.GETTING_STARTED_ON_THE_DEVOPS_PLATFORM,
   );
 
-  const sortedTopics = topicsWithoutDSAndFC.sort((topicA, topicB) => {
+  const designSystem = { ...topics[designIndex] };
+  const featuredResources = { ...topics[featuredIndex] };
+  const devOpsPlatform = { ...topics[devOpsPlatformIndex] };
+  // remove selected topics
+  const topicsWithoutSelectedTopics = topics
+    .filter(topic => topic.node.name !== TOPICS.FEATURED_RESOURCES)
```

Comment:
this _could_ be combined into one filter, prevents the loop from running 3x. But I think if we were talking Big (O) this would all still be O(n) right?

Reply:
yeah still O(n), but it will look cleaner that way